android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
```

Comment:
Q: What's this for? Is this the IP of the device?

Reply:
The IP address of simulators